warehouse/manage/views.py

```diff
@@ -683,7 +685,25 @@ def change_project_role(project, request, _form_class=ChangeRoleForm):
                             submitted_from=request.remote_addr,
                         ),
                     )
+                    owners = (
+                        request.db.query(Role)
+                        .join(Role.user)
+                        .filter(
+                            Role.role_name == 'Owner',
+                            Role.project == role.project,
+                        )
+                    )
+                    owner_emails = [owner.user.email for owner in owners
+                                    if owner.user.email]
+
                     role.role_name = form.role_name.data
+                    send_user_role_changed_email(request, role)
+                    if owner_emails:
```

Comment:
Can we move this condition into the `send_role_changed_for_user_email` function instead?